crates/bevy_app/src/app.rs

```diff
@@ -397,9 +397,26 @@ impl App {
     /// # fn system_a() {}
     /// # fn system_b() {}
     /// # fn system_c() {}
+    /// # fn system_d() {}
     /// #
     /// app.add_systems((system_a, system_b, system_c));
+    /// app.add_systems((system_a, system_b, system_c).chain());
+    /// app.add_systems((system_a, system_b, system_c).after(system_d));
     /// ```
+    ///
+    /// # Note
+    ///
+    /// Using [`in_set`](`IntoSystemConfigs<P>::in_set`),
+    /// [`in_base_set`](`IntoSystemConfigs<P>::in_base_set`),
+    /// [`before`](`IntoSystemConfigs<P>::before`),
+    /// [`after`](`IntoSystemConfigs<P>::after`),
+    /// [`run_if`](`IntoSystemConfigs<P>::run_if`),
+    /// [`ambiguous_with`](`IntoSystemConfigs<P>::ambiguous_with`)
+    /// or [`ambiguous_with_all`](`IntoSystemConfigs<P>::ambiguous_with_all`)
+    /// on [`IntoSystemConfigs<P>`] like in the third example above will implicitly add an
+    /// [`AnonymousSet`] that contains all listed systems.
+    ///
+    /// [`AnonymousSet`]: ../ecs/schedule/struct.AnonymousSet.html
```

Comment:
Im not sure about the `[AnonymousSet]: ../ecs/schedule/struct.AnonymousSet.html` line, but this was the only way I could get my local rustdoc to produce a correct link without having to import the otherwise unused `AnonymousSet` struct. Maybe someone has an better idea?

Reply:
bevy_ecs::schedule::AnonymousSet should work.

Reply:
Yeah, I've tried that and rust-analyzer resolves the link correctly, but in the html output of rustdoc an incorrect `<a href="bevy_ecs::schedule::AnonymousSet">` is produced. But maybe this is only an issue when running rustdoc locally. So I changed it to your suggestion, thank you.

Reply:
Investigating further, the links in docs.rs and the internal dev-docs are also wrong. See the links in https://docs.rs/bevy/0.9.1/bevy/app/struct.App.html#method.register_type as an example.

Reply:
Oof. That looks like a rustdoc bug? I'd probably leave it as is: people will get the idea and as things get improved these links will all get fixed.

Reply:
Okay.